app/assets/javascripts/utils/course_date_utils.js

```diff
@@ -127,6 +127,14 @@ const CourseDateUtils = {
     const timelineStart = startOfWeek(toDate(course.timeline_start));
     return differenceInWeeks(timelineStart, courseStart);
   },
+  getWeekStartDate(week, timelineStart) {
```

Comment:
It looks like these functions are not used?

Reply:
Yes, these are related to the initial commit and this redundant. I will remove these.

Reply:
Thanks. The issue from my January 18 comment remains: this implements new code to calculate the dates within the Milestone component, rather than refactoring how and where the dates are calculated so that the same calculation can be used for both the Milestones and the Timeline.

Reply:
@ragesoss I used the calculating timeline dates code logic in week.jsx 
![image](https://github.com/WikiEducationFoundation/WikiEduDashboard/assets/122573982/747e3726-bd7a-4431-a388-bcc999b064cf)
and implemented dateCalc in milestones.jsx, should I just implement a dateCalc function in dateCalculator.jsx and use that to calculate dates for both?

Reply:
@ragesoss A similar logic is being used in week.jsx as well as empty_week.jsx so a function being implemented as a util would make more sense to be in sync for all files. What do you think?